### ISSUE TITLE
Show tooltip on click as well as hover

### ISSUE THREAD

**Issue by ronvoluted** [StephanWagner/svgMap#14]
Since there are no real hover events on touchscreens, a simple way to get tooltips working on mobile would be to show them on click/touch.

**Comment by StephanWagner** [comment]
Yeah, I had the idea too, just didn’t get around to implement that yet.

--
Stephan Wagner
Philipp-Häring-Str. 8
86157 Augsburg

+49 176 45783473
https://stephanwagner.me

**Comment by ronCYA** [comment]
Good things come to those who wait 👍
